### PR TITLE
Many-to-many relationship table project module 4

### PR DESCRIPTION
**Pull request by juaryimami**
### In this project,
---
- I added some many-to-many relationships and wrote more complex queries to use them to answer questions.
- I used `JOIN`,`INNER JOIN`, `RIGHT JOIN` conjecture to solve the problems.
- The screenshot of my query results are shown below.
----
Screenshot from 2023-01-19 17-28-05
Screenshot from 2023-01-19 17-40-28
Screenshot from 2023-01-19 18-28-05
Screenshot from 2023-01-20 10-48-01
Screenshot from 2023-01-20 15-16-24
Screenshot from 2023-01-20 15-17-20
Screenshot from 2023-01-20 15-21-29
Screenshot from 2023-01-20 15-29-59
Screenshot from 2023-01-20 15-39-58
Screenshot from 2023-01-20 15-47-26
Screenshot from 2023-01-20 15-53-36
Screenshot from 2023-01-20 16-00-38
Screenshot from 2023-01-20 16-07-55

---
#### Thank you for reviewing my project!!!!!!!.

### CODE REVIEW

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @juaryimami, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: data.sql [hunk 1]
```diff
@@ -32,4 +32,34 @@ INSERT INTO  owners(full_name,age) VALUES('Dean Whinchester',14);
...
+
+-- insertion to specialites table
+
+
+-- insertion to visits table
+insert into visits(vets_id,animal_id,date_of_visit) values(1,1,'2020-05-24');
+insert into visits(vets_id,animal_id,date_of_visit) values(3,1,'2020-07-22');
+insert into visits(vets_id,animal_id,date_of_visit) values(4,2,'2021-02-02');
+insert into visits(vets_id,animal_id,date_of_visit) values(2,3,'2020-01-05');
+insert into visits(vets_id,animal_id,date_of_visit) values(2,3,'2020-05-14');
```
Comment (DeliceLydia) [inline data.sql]: - [ ] Please add provided data in the `specialization` table

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @juaryimami, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: data.sql (hunk 1 above)
Comment (juaryimami) [reply data.sql]: ohhh ok ok sorry

**Review by rustam-tolipov — APPROVED** [review]
Hi @juaryimami, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
Kindly add the MIT license to your readme :)

Also, you are free to remove this line from your readme:

> NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — COMMENTED** [review]
Hi @juaryimami 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Good looking readme file 👍 
* Well written PR 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**